src/lib/eina/eina_inline_lock_win32.x

```diff
@@ -461,10 +460,8 @@ _eina_tls_get(Eina_TLS key)
 static inline Eina_Bool
 _eina_tls_set(Eina_TLS key, const void *data)
 {
-   void *p = TlsGetValue(key);
    const Eina_Bool ret = TlsSetValue(key, (void *) data) ? EINA_TRUE : EINA_FALSE;
    eina_lock_take(&_eina_tls_map_lock);
-   if (ret) _eina_free_tls_value(&key, p);
    eina_lock_release(&_eina_tls_map_lock);
```

Comment:
Do we still need this lock/release than?

Reply:
You are right, fixed. Thanks.